openmeetings-core/src/main/java/org/apache/openmeetings/core/remote/KStream.java

```diff
@@ -504,9 +513,11 @@ public void onError(Throwable cause) throws Exception {
 		sipProcessor = Optional.empty();
 	}
 
-	public void addCandidate(IceCandidate candidate, String uid) {
+	public void addIceCandidate(IceCandidate candidate, String uid) {
 		if (this.uid.equals(uid)) {
 			if (!(outgoingMedia instanceof WebRtcEndpoint)) {
```

Comment:
this might be bad idea :(
`outgoingMedia` can be `null` in case if it is too early
and it might be `RtpEndpoint` in case `sipClient == true`

in latter case candidate shouldn't go to the queue

Reply:
outgoingMedia can be null in case if it is too early
=> yeah, we would need to queue them and apply them after outgoingMedia is no more null.

Reply:
It seems I wasn't clear enough :((
`outgoingMedia` can be

1. `null`
2. `WebRtcEndpoint`
3. `RtpEndpoint`

`!(outgoingMedia instanceof WebRtcEndpoint)` will return `true` in cases `1 and 3`

ice candidated should be queued only in case `1`